packages/shared/src/components/squads/SquadListingHeader.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactElement } from 'react';
+import { cloudinary } from '../../lib/image';
+import { Button } from '../buttons/Button';
+import { squadsPublicWaitlist } from '../../lib/constants';
+import SourceBetaIcon from '../../../icons/source_beta.svg';
+import { Origin } from '../../lib/analytics';
+import { useSquadNavigation } from '../../hooks';
+
+interface SquadListingHeaderProps {
+  isOwner?: boolean;
+}
+
+export const SquadListingHeader = ({
+  isOwner = false,
+}: SquadListingHeaderProps): ReactElement => {
+  const { openNewSquad } = useSquadNavigation();
+
+  return (
+    <div className="mb-4">
+      <div
+        className="flex flex-col items-center p-6 mb-1 text-center bg-center bg-cover rounded-24 bg-theme-bg-primary"
+        style={{
+          backgroundImage: `url(${cloudinary.squads.directory.banner})`,
+        }}
+      >
+        <div className="mb-4">
+          <SourceBetaIcon width="auto" height="3rem" className="text-white" />
+        </div>
```

Comment:
Is it possible to avoid the container?
```suggestion
      <SourceBetaIcon width="auto" height="3rem" className="text-white mb-4" />
```